zuul-core/src/main/java/com/netflix/zuul/netty/server/Server.java

```diff
@@ -254,7 +256,14 @@ private ChannelFuture setupServerBootstrap(
         serverStatusManager.localStatus(InstanceInfo.InstanceStatus.UP);
 
         // Bind and start to accept incoming connections.
-        return serverBootstrap.bind(listenAddress).sync();
+        ChannelFuture bindFuture = serverBootstrap.bind(listenAddress);
+        try {
+            return bindFuture.sync();
+        } catch (Exception e) {
```

Comment:
Could you catch a more specific exception?

Reply:
I can't because the sync method doesn't actually declare it.   It's a compile error to catch something more specific.